OKTA-535455 - Improved docs for per OAuth 2.0 app client rate limiting (#5389)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

#### Event count graphs

> The event count graph plots the number of violations, bursts, and warnings over the selected time period, which can be the last 24 hours, the last 7 days, last month, or the last 3 months. The events per API bar graph shows the top five number of events by API over the same time period.

Reviewer: ```suggestion
The event count graph plots the number of violations, bursts, and warnings over the selected time period. This can be the last 24 hours, the last seven days, last month, or the last three months. The events per API bar graph shows the top five number of events by API over the same time period.
```

Author: I'm not a fan of using "This" in this way, unclear antecedent rule or something like that? I.e., it's not clear what "this" can be. But because it refers to a "time period", which is at the end of the previous sentence, I can see that as being okay and not necessary to clarify. But I find the first construction reads better, and if we're only changing it to drop the sentence count down, I'd rather keep it that way. (changing numbers to their written equivalent is fine, although I have some radical thoughts about that classic rule, too.)

</div>

> You can sort on all the columns. The trendline is sorted by the current usage. Use the **Filters** dropdown menu to filter between all APIs, those that have rate-limit multipliers applied or are multiplier eligible, or those that are ineligible.

Reviewer: ```suggestion
You can sort on all the columns. The trendline is sorted by the current use. Use the **Filters** dropdown menu to filter between all APIs, those that have rate-limit multipliers applied or are multiplier eligible, or those that are ineligible.
```

</div>

> The key benefit of purchasing the rate limit multiplier, therefore getting an increased rate limit, isn't to stop end users from using an app. The rate limit multiplier is based on the customer's org and what the expected use would be. If a customer has dynamic scale, the info icon beside **Rate Limit** in the overview pane indicates which tier was purchased, and the graphs in the [Rate limit usage over time pane](#rate-limit-usage-over-time-pane) reflect the applied rate limit multiplier. To purchase the rate limit multiplier, either create a support ticket or contact your customer support representative.

Reviewer: ```suggestion
The key benefit of purchasing the rate limit multiplier, therefore getting an increased rate limit, isn't to stop end users from using an app. The rate limit multiplier is based on the customer's org and expected use. If a customer has dynamic scale, the info icon beside **Rate Limit** in the overview pane indicates which tier was purchased, and the graphs in the [Rate limit usage over time pane](#rate-limit-usage-over-time-pane) reflect the applied rate limit multiplier. To purchase the rate limit multiplier, either create a support ticket or contact your customer support representative.
```

#### Bar graph view

The bar graph compares a rogue IP or app with other IPs or apps. For example, the bar graph can show a single rogue IP that’s responsible for a spike in traffic that appears in the line graph. It can also show that many users are signing in at the same time, perhaps to attend a company-wide meeting or to check out a special offer that was delivered in a mass email to a company's clients.

### Potential next steps

If you find that the rate limit use is high, you may want to adjust your rate limit maximums. If your company is planning an upcoming event, such as an upcoming marketing campaign, you may want to figure out the rate limit capacity that you'd need. In both cases, Okta recommends that you contact Okta Support to determine and adjust your rate limits. See [How to Request a Temporary Rate Limit Increase](https://support.okta.com/help/s/article/How-can-we-request-to-have-the-rate-limit-for-our-org-temporarily-increased?language=en_US).

> If the usage is high over a long period, you may need to lower the usage. This means that if high traffic usage points to a particular offender, you would need to inform or block the offender (such as a malfunctioning app, a bad script, or a rogue user) that’s causing the traffic. For example, you can either reach out to the team that writes the script or block the user or app that’s causing the traffic.

Reviewer: ```suggestion
If the use is high over a long period, you may need to lower the use. This means that if high traffic use points to a particular offender, you would need to inform or block the offender (such as a malfunctioning app, a bad script, or a rogue user) that’s causing the traffic. For example, you can either reach out to the team that writes the script or block the user or app that’s causing the traffic.
```
